controllers/DataAnalysisController.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+//******************************************************************************
+//                                       DataAnalysisController.php
+// SILEX-PHIS
+// Copyright © INRA 2018
+// Creation date: 21 feb 2019
+// Contact: arnaud.charleroy@inra.fr, anne.tireau@inra.fr, pascal.neveu@inra.fr
+//******************************************************************************
+
+namespace app\controllers;
+
+use Yii;
+use yii\filters\VerbFilter;
+use app\models\yiiModels\DataAnalysisAppSearch;
+
+include_once '../config/web_services.php';
+
+/**
+ * Implements the controller for available statistical and visualisation programs
+ * @see yii\web\Controller
+ * @author Arnaud Charleroy <arnaud.charleroy@inra.fr>
+ */
+class DataAnalysisController extends \yii\web\Controller {
+    /**
+     * Constants used to parse configuration file
+     */
+    const INTEGRATED_FUNCTIONS = "integratedFunctions";
+    const FORM_PARAMETERS = "formParameters";
+    
+    /**
+     * define the behaviors
+     * @return array
+     */
+    public function behaviors() {
+        return [
+            'verbs' => [
+                'class' => VerbFilter::className(),
+                'actions' => [
+                    'delete' => ['POST'],
+                ]
+            ]
+        ];
+    }
+
+    /**
+     * List all R apps
+     * @return mixed
+     */
+    public function actionIndex($integrated = false) {
```

Comment:
The documentation for the parameter $integrated is missing

Reply:
Done.